website/src/components/SearchPage/DownloadDialog/DownloadUrlGenerator.ts

```diff
@@ -53,7 +53,7 @@ export class DownloadUrlGenerator {
         }
 
         downloadParameters.toUrlSearchParams().forEach(([name, value]) => {
-            if (value.length > 0) {
+            if (value.length && !params.has(name)) {
```

Comment:
Nope, this rightly makes tests fail as it would mess up the entry of multiple accessionVersions